### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by MrBretze** [haskellcamargo/skype-unofficial-client#50]
Hello do you want to port skype-unofficial-client on Windows, (https://github.com/Aluxian/Facebook-Messenger-Desktop) use gulp to compile :/ it is juste a idea :/

**Comment by Delta-40** [comment]
Why would you use this on windows? Windows have a perfectly application already.

**Comment by MrBretze** [comment]
Because I prefer the visual of the web version

**Comment by Delta-40** [comment]
Visual, they are exactly the same. Have you updated your windows lately?

**Comment by MrBretze** [comment]
Yes i have windows 10 ans the last version of skype and nop the web version is not exactly the same of app version

**Comment by Delta-40** [comment]
Try update skype. They are exactly the same, the windows desktop app is even MUCH better (notifications, P2P-groups etc) than the web version here.

**Comment by MrBretze** [comment]
Web: http://prntscr.com/b6q99b
App: http://prntscr.com/b6q9gk

For me is not the same icon and i prefer the list of contact in the web version

**Comment by Kuchiriel** [comment]
https://github.com/nwjs/nw.js/wiki/Application-auto-building-scripts-for-Linux-Windows-and-OSX

**Comment by maxdeepfield** [comment]
> Why would you use this on windows? Windows have a perfectly application 

Starting february 2018 official skype desktop app is automatically deleted on Windows 10 machines, they say we need to use that crappy something we can find in Store. Now personally I prefer web skype instead. Also all the skypes (mobile and desktop) have weird bugs with message delivery, only web skype shows all the messages.

**Comment by haskellcamargo** [comment]
@Kuchiriel do you believe it is time to resurrect the project?

**Comment by maxdeepfield** [comment]
already posted on ghetto-skype page, sorry

11 мар. 2018 г. 1:38 пользователь "Marcelo Camargo" <
notifications@github.com> написал:

**Comment by Kuchiriel** [comment]
@haskellcamargo, We can talk about it, but we need to set some standards to make a good project.

**Comment by onlurking** [comment]
I think this will need a major rewrite/refactor to have feature parity with the [ghetto-skype](https://github.com/stanfieldr/ghetto-skype).

**Comment by maxdeepfield** [comment]
web skype also have problems with message delivery. think Windows support can be closed as ANY skype related projects.
